packages/core/getBusyTimes.ts

```diff
@@ -67,6 +69,9 @@ export async function getBusyTimes(params: {
     status: {
       in: [BookingStatus.ACCEPTED],
     },
+    uid: {
+      not: rescheduleBookingId, // We dont want to fetch the booking that is being rescheduled - This allows us to reschedule for the timeslot that the existing booking takes up
```

Comment:
I am getting an error when I am trying to book the same slot again, `rescheduleBookingId` is undefined in this case
![Screenshot 2023-04-07 at 09 47 42](https://user-images.githubusercontent.com/30310907/230569047-003c78f5-b003-4952-96b1-b705fe1ec298.png)